InnerEye/ML/run_ml.py

```diff
@@ -740,6 +740,7 @@ def are_sibling_runs_finished(self) -> bool:
         else:
             raise NotImplementedError("are_sibling_runs_finished only works for cross validation runs in AzureML.")
 
+    @torch.no_grad()
```

Comment:
why do you have the no_grad here, when you already have it on `predict_whole_image`? What is the offending operation, is it computing the output shape? Will adding it there solve the problem too? (Saying this just because I feel that the decorator should be added at the innermost level where necessary)

Reply:
Will dig further into this